cog/daily_charge.py

```
@@ -23,95 +25,293 @@
+
+    # TODO: inherit a MySQLCursorAbstract to add method about these or consider to add self.cursor
+    @staticmethod
+    def reward(
+        user_or_uid: _UserTag | int,
+        last_charge: datetime,
+        executed_at: datetime,
+        orig_combo: int,
+        orig_point: int,
+        orig_ticket: int,
+        cursor: MySQLCursorAbstract | None = None,
+        is_forgivable: bool = False,
```

Ruff (0.8.2)

Boolean-typed positional argument in function definition

(FBT001)